specification/cognitiveservices/data-plane/Personalizer/preview/2022-09-01-preview/Personalizer.json

```diff
@@ -13,7 +13,7 @@
     "/configurations/service": {
       "get": {
         "tags": [
-          "ConfigurationsV1Dot1Preview1"
+          "Configurations20220901Preview"
```

Comment:
why we had versioning in tag?

Reply:
The swagger generation process generates the tag this way

Reply:
As Travis said, so far we are not aware any tooling using the "tags" field. So not an objection here.

However, it still perplexes human reader. And we do not recommend using automatically generated swagger (e.g. via Swashbuckle).